nipoppy/trackers/fmriprep_tracker.py

```diff
@@ -1,11 +1,7 @@
 from pathlib import Path
 import os
 
-# Status flags
-SUCCESS="SUCCESS"
-FAIL="FAIL"
-INCOMPLETE="INCOMPLETE"
-UNAVAILABLE="UNAVAILABLE"
+from trackers.tracker import SUCCESS, FAIL
```

Comment:
If I am remembering this correctly, we went back to having more options such as `INCOMPLETE`, `UNAVAILABLE`?

Reply:
I think we dropped `INCOMPLETE` but still have `UNAVAILABLE`

Reply:
`UNAVAILABLE` is only used in `run_tracker.py`